FoodOrderingApp-service/src/main/java/com/upgrad/FoodOrderingApp/service/entity/CustomerAddress.java

```diff
@@ -0,0 +1,55 @@
+package com.upgrad.FoodOrderingApp.service.entity;
+
+//import org.apache.commons.lang3.builder.ToStringExclude;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.io.Serializable;
+
+/**
+ * CustomerAddress class contains all the attributes to be mapped to all the fields in 'customer_address' table in the database
+ */
+
+
+public class CustomerAddress implements  Serializable {
```

Comment:
You have already added `AddressEntity` do we need this second one? I don't see this being used anywhere.

Reply:
I will delete it for now. It might be used in other endpoints, will check it and delete it.